var/spack/repos/builtin/packages/py-html5lib/package.py

```diff
@@ -12,12 +12,19 @@ class PyHtml5lib(PythonPackage):
     homepage = "https://github.com/html5lib/html5lib-python"
     pypi = "html5lib/html5lib-099.tar.gz"
```

Comment:
```suggestion
    pypi = "html5lib/html5lib-1.1.tar.gz"
```
If you make this change, it will fix the URL parsing issue and you won't need to use `url_for_version`